client/data/media/use-add-media.js

```diff
@@ -1,12 +1,39 @@
 import { useCallback } from 'react';
 import { getFileUploader } from 'calypso/lib/media/utils';
+import { isFileList } from 'calypso/state/media/utils/is-file-list';
 import { useUploadMediaMutation } from './use-upload-media-mutation';
 
 export const useAddMedia = () => {
 	const { uploadMediaAsync } = useUploadMediaMutation();
+	const addVideopressStatusToFile = ( file, site ) => {
+		const isSiteJetpack = !! site.jetpack;
+		const isVideoPressEnabled = site.options && site.options.videopress_enabled;
+		const isVideoPressModuleActive =
+			! isSiteJetpack ||
+			( site.options.active_modules && site.options.active_modules.includes( 'videopress' ) );
+		const canUseVideopress = isVideoPressEnabled || isVideoPressModuleActive;
+
+		const fileCallback = ( fileObject ) => ( fileObject.canUseVideopress = canUseVideopress );
```

Comment:
This line is confusing to look at... should it be using `{` instead of `(`? Or is this what the linter wanted?

Reply:
It's technically a one-liner arrow function, I added parenthesis for readability, so we know it's in the function body, not a weird double assignment, as there's also an equal sign for fileCallback.
Let's say it fails at this purpose if you were confused by it, I'll update it 😄